flake.nix

```diff
@@ -33,10 +33,17 @@
           crypto3-debug-tests = (pkgs.callPackage ./crypto3.nix {
             enableDebug = true;
             runTests = true;
+            sanitize = true;
```

Comment:
why sanitize turned on for just debug test configuration?

Reply:
Reverted